spec/controllers/ems_common_controller_spec.rb

```diff
@@ -130,5 +130,36 @@
         ManageIQ::Providers::Kubernetes::ContainerManager.last.authentication_token("bearer").should == "valid-token"
       end
     end
+
+    context "#button" do
+      before(:each) do
+        set_user_privileges
+        FactoryGirl.create(:vmdb_database)
+        EvmSpecHelper.create_guid_miq_server_zone
+      end
+
+      it "when VM Migrate is pressed for unsupported type" do
+        controller.stub(:role_allows).and_return(true)
+        controller.stub(:performed?).and_return(true)
```

Comment:
@martinpovolny when running spec test for some reason performed? was always returning false. Maybe i am doing something incorrectly, i will look again.